cmd/webhook/init/server/server_test.go

```diff
@@ -77,39 +77,14 @@ func TestRecords(t *testing.T) {
 			body:               "",
 			expectedStatusCode: http.StatusOK,
 			expectedResponseHeaders: map[string]string{
-				"Content-Type": "application/external.dns.webhook+json;version=1",
+				"Content-Type": api.MediaTypeFormatAndVersion,
 			},
 			expectedBody: "[{\"dnsName\":\"test.example.com\",\"targets\":[\"\"],\"recordType\":\"A\",\"recordTTL\":3600,\"labels\":{\"label1\":\"value1\"}}]",
 		},
-		{
```

Comment:
the reason I removed those cases is that the official webhooks server is not doing any checks on the accept header: 
https://github.com/kubernetes-sigs/external-dns/blob/master/provider/webhook/api/httpapi.go#L45
https://github.com/kubernetes-sigs/external-dns/blob/master/provider/webhook/api/httpapi.go#L81

I guess the accept header are mainly for information purpose.